news/_posts/2022-04-12-cyot.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "New Tutorial Feature: Choose Your Own Tutorial"
```

Comment:
Also probably it would be better to include it in a different PR.

Reply:
I put it into #3389 but not sure it was the best way to do it.

Reply:
It's fine for me either way. It was a feature specifically implemented just for this PR, so, it feels less important to split it out. But since @lldelisle opened the PR we can merge that as well and rebase this to remove those commits (but it feels like a lot of work.)

Reply:
If it is related and @hexylena agrees, then it is fine.